cardano-node.cabal

```diff
@@ -11,43 +11,70 @@ cabal-version:  >= 1.10
 extra-source-files: README.md, ChangeLog.md
 
 library
+  hs-source-dirs:      src
+
   exposed-modules:
-      Lib
+                       Lib
+
   other-modules:
-      Paths_cardano_node
-  hs-source-dirs:
-      src
+                       Paths_cardano_node
+
   build-depends:
-      base >=4.7 && <5
-    , cardano-ledger
-    , cardano-prelude
-    , cardano-shell
-    , ouroboros-consensus
-    , ouroboros-network
-  default-language: Haskell2010
+                       base >=4.7 && <5
+                     , cardano-ledger
+                     , cardano-prelude
+                     , cardano-shell
+                     , ouroboros-consensus
+                     , ouroboros-network
+
+  default-language:    Haskell2010
+  default-extensions:  NoImplicitPrelude
+
+  ghc-options:         -Weverything
+                       -fno-warn-safe
+                       -fno-warn-unsafe
+                       -fno-warn-missing-import-lists
 
 executable cardano-node-exe
-  main-is: Main.hs
+  main-is:              Main.hs
```

Comment:
Since this is the main Haskell file for an executable, it does not need to be camel case. All lower case would work.

Also, why not name it `cardano-node.hs` since that is the name of executable its going to create.

Finally, can we drop the "-exe" from the executable's name?